NitroxClient/GameLogic/InitialSync/PdaInitialSyncProcessor.cs

```diff
@@ -89,7 +89,7 @@ private void SetKnownTech(List<NitroxTechType> techTypes)
             {
                 foreach (NitroxTechType techType in techTypes)
                 {
-                    HashSet<TechType> complete = (HashSet<TechType>)(typeof(PDAScanner).GetField("complete", BindingFlags.NonPublic | BindingFlags.Static).GetValue(null));
+                    //HashSet<TechType> complete = (HashSet<TechType>)(typeof(PDAScanner).GetField("complete", BindingFlags.NonPublic | BindingFlags.Static).GetValue(null));
```

Comment:
I don't know, it's just unused so I don't know why it's there. Maybe delete it?

Reply:
Looks like it has been unused since its conception?

https://github.com/SubnauticaNitrox/Nitrox/pull/246/files#diff-a0d0db28bd0e1a4228654e8735f21ef1R57